### ISSUE TITLE
How do I authorize myself to use DELETE/POST/PUT endpoints?

### ISSUE THREAD

**Issue by Blackpillar** [plamere/spotipy#770]
I am very new to all of this authorization stuff and I couldn't find a newbie guide to walk me through the process of authorization to use the aforementioned endpoints.
So how would I be able to authenticate myself to use said endpoints using spotipy?

Thanks in advance!

**Comment by stephanebruckert** [comment]
Not saying the docs couldn't be simplified, but everything should be there https://spotipy.readthedocs.io/en/2.19.0/#getting-started

Feedback on how you would improve is very welcome!

**Comment by Blackpillar** [comment]
Thanks for replying, as a complete beginner, I didn't know how to use scopes, especially when the link in the docs that's supposed to send you to them returns a 404 error. Only after a bit of error searching I was able to understand how to use them and then realized how easy it all is.
(Where I found the answer I needed https://stackoverflow.com/questions/56173066/how-to-solve-insufficient-client-scope-in-python-using-spotipy)

**Comment by stephanebruckert** [comment]
Good catch on finding the dead link! If you want, you can contribute to this repo and fix the doc. This is where the documentation code is:

https://github.com/plamere/spotipy/blame/07915e21e558b6b867fc00cfffc0d9b06d3266e7/docs/index.rst#L146

There is another reference to that link here:

https://github.com/plamere/spotipy/blame/master/FAQ.md#L39

**Comment by Blackpillar** [comment]
Sorry I ended up requesting two changes on the same thing, I mistakengly pasted the web archives link. But my latest change request is the correct link.

Thanks for the help!

**Comment by stephanebruckert** [comment]
I can't see any new pull request, did you create it https://github.com/plamere/spotipy?

**Comment by Blackpillar** [comment]
Yes I did create it there, but then again, this is my first time making a pull request, so I am probably doing something wrong.

Here is what I did:
1) I went to https://github.com/plamere/spotipy
2) I went to docs > index.rst
3) I selected the v3 branche
4) I pressed "Edit the file in your fork of this project"
5) Made the change and clicked "Propose changes"

Now that I think about it, the 4th step is probably what I did wrong. Here is the [correct link](https://developer.spotify.com/documentation/general/guides/authorization/scopes/) if you want to fix it yourself, but I'd appreciate if you show me how I can contribute.

**Comment by stephanebruckert** [comment]
> 3. I selected the v3 branche

Ha yes, don't select the v3 branch but stay on `master`

**Comment by Blackpillar** [comment]
The master branche has the correct link, the V3 one doesn't, so I figured https://spotipy.readthedocs.io/en/2.19.0/ is using V3 index. So now I am bit confused on what's going on 😖

**Comment by stephanebruckert** [comment]
No https://spotipy.readthedocs.io/en/2.19.0/ is using v2.19.0 (master). Just ignore v3 as it's not released yet. Also changes are pulled from v2 to v3, not the opposite. So your changes will end up in v3 too at some point!

**Comment by Blackpillar** [comment]
But the master branche's index.srt is using the correct scopes link, but when you go to https://spotipy.readthedocs.io/en/2.19.0/ and click the link, it takes you the link that V3 has.

**Comment by stephanebruckert** [comment]
Oh you are right, actually the link was fixed on master https://github.com/plamere/spotipy/pull/738 3 months ago and the reason it's not reflected in the doc is that no new version was published. I will try to do that soon... Feel free to ignore that now, but thanks a lot for trying!
